mysql/init.sql

```diff
@@ -0,0 +1,10 @@
+GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
+
+-- Use the 'app' database
+USE app_db;
+
+-- Create a table to store counter data
+CREATE TABLE counter (
+   id INT AUTO_INCREMENT PRIMARY KEY,
+   value INT
+);
```

Comment:
No need to create extra initialization file for the database, it should be done by app